server/controllers/tasks/lib/check_entity.coffee

```diff
@@ -0,0 +1,48 @@
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+promises_ = __.require 'lib', 'promises'
+error_ = __.require 'lib', 'error/error'
+
+searchWikidataByText = __.require 'data', 'wikidata/search_by_text'
+
+Task = __.require 'models', 'task'
+
+module.exports = (entity)->
+  title = _.values(entity.labels)[0]
+
+  searchWikidataByText
+    search: title
+  .then (searchResult)->
+    entities = _.values(searchResult.entities)
+    filterSuggestions entity, entities
+  .then (entities)->
+    tooManyHomonyms(entities)
+  .then (suggestionEntities)->
+    newTasks = createTasks entity, suggestionEntities
+    return { "tasks": newTasks }
```

Comment:
so now that we have a lib to generate tasks from an entity, what we need is a script that would:
- get entities that it would need to be checked: this could rely on another lib that would take of fetching entities that aren't yet targeted by a task
- pass those entities to `checkEntity` and pass the generated tasks docs to another lib function (probably in `controllers/tasks/lib/tasks.coffee`) to post it in `tasks` database (which btw needs to be added to [database list](https://github.com/inventaire/inventaire/blob/3258a58/server/db/couch/list.coffee))

This script could then be run periodically (as a system cron task maybe?) to feed the tasks database. We would need then to make those tasks accessible (`GET`) and updatable (`PUT`) from the client, by creating an `/api/tasks` endpoint, but we will get to that later.